src/PostCampaign.php

```diff
@@ -552,7 +552,7 @@ private static function migrate_old_vars( array $css_vars ): array {
 				continue;
 			}
 			foreach ( $to as $new_name ) {
-				$css_vars[ $new_name ] = $css_vars[ $from ];
+				$css_vars[ '--' . $new_name ] = $css_vars[ $from ];
```

Comment:
Quick fix because otherwise it puts some variables in there twice. In case of new themes `$css_vars` will contain the version with `--` in front. Without this fix it would add it a second time without 2 dashes. In the template it's currently set up to work with both variants.

Will look for a cleaner way to handle this.

Reply:
Addressing this in https://github.com/greenpeace/planet4-master-theme/pull/1428/files